### ISSUE TITLE
ESQL: duplicated aggregates not allowed

### ISSUE THREAD

**Issue by astefan** [elastic/elasticsearch#99888]
### Description

A query like `from employees | keep salary, salary, first_name | limit 1` (where a field is specified multiple times) allows for the same column name to be used multiple times, with a result like

```
    salary     |    salary     |  first_name   
---------------+---------------+---------------
57305          |57305          |Georgi         
```

but, the same "type" of approach, but for `stats` doesn't work unless specific aliasing is used. So, a query like `from employees | stats max(salary), min(salary), min(salary), max(salary)` generates an output like

```
  min(salary)  |  max(salary)  
---------------+---------------
25324          |74999          
```

why not 4 columns?
If, on the other hand, specific aliasing is used - `from employees | stats m1 = max(salary), m2 = min(salary), m3 = min(salary), m4 = max(salary)` the result will have the 4 desired columns:

```
      m1       |      m2       |      m3       |      m4       
---------------+---------------+---------------+---------------
74999          |25324          |25324          |74999          
```

In conclusion, if `keep` allows same column multiple times in the results, so should `stats`. For consistency.

**Comment by costin** [comment]
Fixed by #99936
